src/index.ts

```diff
@@ -94,6 +94,12 @@ export default class BugzillaAPI {
     );
   }
 
+  public getBug(id: number | string): FilteredQuery<Bug> {
```

Comment:
The return type for this method is incorrect, `FilteredQuery` resolves to an array so it is correct for `getBugs` but won't work here.